paper/paper.md

```diff
@@ -8,32 +8,36 @@ authors:
+![The logo of insight represents a statistical model, in the metaphorical form of a monster, which internals are being exposed, summarizing the aim and philosophy underlying the package.](figure1.png)
+
+The *insight* package fills this important gap by providing a suite of functions to support almost any model (see a list of the many models supported below in the **Supported Models** section). The goal of *insight*, then, is to provide tools to provide *easy*, *intuitive*, and *consistent* access to information contained in model objects. Ultimately, the development of *insight* is in line with the overall philosophy of the [easystats project](https://github.com/easystats), which is to facilitate and streamline the process of doing statistical analysis and reporting the results in the R programming language. 
 
 Built with non-programmers in mind, *insight* offers a broad toolbox for making model and data information easily accessible, revolving around two key prefixes: `get_*` and `find_*`. Generally, the `get_*` prefix extracts values associated with model-specific objects (e.g., parameters or algorithms), while the `find_*` prefix lists model-specific objects (e.g., priors and predictors). In total, the *insight* package includes 16 core functions: `get_data()`, `get_priors()`, `get_variance()`, `get_parameters()`, `get_predictors()`, `get_random()`, `get_response()`, `find_algorithm()`, `find_formula()`, `find_variables()`, `find_terms()`, `find_parameters()`, `find_predictors()`, `find_random()`, `find_response()`, and `model_info()`. In all cases, users must supply at a minimum, the name of the model fit object. In several functions, there are additional arguments that allow for more targeted returns of model information. For example, the `find_terms()` function's `effects` argument allows for the extraction of "fixed effects" terms, "random effects" terms, or by default, "all" terms in the model object. We point users to the package documentation or the complementary package website, https://easystats.github.io/insight/, for a detailed list of the arguments associated with each function as well as the returned values from each function.
 
 For a more intuitive introduction, consider the following examples for three major types of models: ordinary least squares (OLS) regression, linear mixed effects models, and Bayesian models. Importantly, though only a few functions are included below for the sake of space, users are encouraged to inspect the package documentation for an exhaustive list of package functionality with accompanying examples.
 
+![The functions in insight allow to access different aspects of models, such as the data used for fitting, the parameters of the fitted model or various information about the model.](figure2.png)
```

Comment:
Did you want to italicize the package name each time (*insight*)? Or should we leave it standard font? I don't have a preference here; just wanting to be consistent for the manuscript.

Reply:
This is a tooltip, I think, so it's not necessary here... But we can check the generated PDF later.